src/Components/Artwork/Save.tsx

```diff
@@ -56,15 +62,20 @@ export class SaveButton extends React.Component<SaveProps, SaveState> {
     return isSaved
   }
 
-  @track<SaveProps>(
-    props =>
-      ({
-        action_type: props.artwork.is_saved
-          ? "Removed Artwork"
-          : "Saved Artwork",
-        entity_slug: props.artwork.id,
-      } as Schema.Old)
-  )
+  trackSave = () => {
+    const {
+      tracking,
+      artwork: { is_saved, id, _id },
+    } = this.props
+    const trackingData: SaveTrackingProps = this.props.trackingData || {}
+    const action = is_saved ? "Removed Artwork" : "Saved Artwork"
```

Comment:
Is this "Removed Artwork" will be descriptive enough? Is not that essentially "Removed from saved works" ?

Reply:
Yea, I'm just keeping what was already here being tracked. The addition was the `context_page` coming thru in `trackingData`.